src/transform.ts

```diff
@@ -146,12 +143,12 @@ export interface WindowTransform {
   ignorePeers?: boolean;
 
   /**
-   * The fields to group by.
+   * The names of the data fields to partioin the objects into seprate windows. If not specified, everything will be in a single group.
```

Comment:
The data fields for partitioning the data objects into separate windows.  If unspecified, all data points will be a single group.